src/test/regress/sql/multi_schema_support.sql

```diff
@@ -842,6 +842,16 @@ ALTER TABLE non_existent_schema.non_existent_table SET SCHEMA another_non_existe
 ALTER TABLE existing_schema.non_existent_table SET SCHEMA another_existing_schema;
 ALTER TABLE existing_schema.non_existent_table SET SCHEMA non_existent_schema;
 ALTER TABLE existing_schema.table_set_schema SET SCHEMA non_existent_schema;
+
+
+-- test ALTER TABLE IF EXISTS SET SCHEMA with nonexisting schemas and table
+ALTER TABLE IF EXISTS non_existent_schema.table_set_schema SET SCHEMA another_existing_schema;
+ALTER TABLE IF EXISTS non_existent_schema.non_existent_table SET SCHEMA another_existing_schema;
+ALTER TABLE IF EXISTS non_existent_schema.table_set_schema SET SCHEMA another_non_existent_schema;
+ALTER TABLE IF EXISTS non_existent_schema.non_existent_table SET SCHEMA another_non_existent_schema;
+ALTER TABLE IF EXISTS existing_schema.non_existent_table SET SCHEMA another_existing_schema;
+ALTER TABLE IF EXISTS existing_schema.non_existent_table SET SCHEMA non_existent_schema;
+ALTER TABLE IF EXISTS existing_schema.table_set_schema SET SCHEMA non_existent_schema;
```

Comment:
can we have one without a fully qualified table name?
```SQL
ALTER TABLE IF EXISTS non_existing_table SET SCHEMA existing_schema;
ALTER TABLE IF EXISTS non_existing_table SET SCHEMA non_existing_schema;
```

This will also follow the codepath where the schema is not set on the parsetree.